### PR TITLE
vpc changed in terraform

### PR DESCRIPTION
**Pull request by HHjolany**
VPC configuration in terraform, changed due to migrating the Housing-development account

### CODE REVIEW

**Review by AnnaGolosova — APPROVED** [review]

**Review by sandipchandra — APPROVED** [review]
approved

File: terraform/staging/main.tf [hunk 1]
```diff
@@ -22,7 +22,7 @@ terraform {
 
 data "aws_vpc" "staging_vpc" {
   tags = {
-    Name = "vpc-housing-staging"
+    Name = "housing-dev"
```
Comment (LBHMGeorgieva) [inline terraform/staging/main.tf:25]: Hi Hamid, the change should be only on the development file

File: terraform/production/main.tf [hunk 2]
```diff
@@ -22,7 +22,7 @@ terraform {
 
 data "aws_vpc" "production_vpc" {
   tags = {
-    Name = "vpc-housing-production"
+    Name = "housing-dev"
```
Comment (LBHMGeorgieva) [inline terraform/production/main.tf:25]: Hi Hamid, the change should be only on the development file

**Review by LBHMKeyworth — APPROVED** [review]
Looks good to me.

**Pull request merged**